packages/flutter_tools/lib/src/services.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:path/path.dart' as path;
+import 'package:yaml/yaml.dart';
+
+import 'artifacts.dart';
+
+const String _kFlutterManifestPath = 'flutter.yaml';
+
+dynamic _loadYamlFile(String path) {
+  if (!FileSystemEntity.isFileSync(path))
+    return null;
+  String manifestString = new File(path).readAsStringSync();
+  return loadYaml(manifestString);
+}
+
+Future parseServiceConfigs(
```

Comment:
It seems a bit odd to me that this is populating the variables that are passed in instead of returning a result of some kind. Perhaps adjust the name of the method, or add some docs to clarify?